apps/namadillo/src/App/Governance/ProposalDescription.tsx

```diff
@@ -1,10 +1,21 @@
 import { SkeletonLoading, Stack } from "@namada/components";
 import { Proposal } from "@namada/types";
+import { proposalFamily } from "atoms/proposals";
 import clsx from "clsx";
 import { useAtomValue } from "jotai";
-import { useState } from "react";
+import { ReactNode, useState } from "react";
+import { twMerge } from "tailwind-merge";
 
-import { proposalFamily } from "atoms/proposals";
+const formatLinks = (string: string): ReactNode => {
+  const regex = /(http.*)/;
+  return string.split(regex).map((part, index) =>
+    regex.test(part) ?
+      <a key={index} href={part} rel="noreferrer nofollow">
```

Comment:
Can you add a target="_blank" here as well? :)

Reply:
Great observation! Thanks!